src/dock.js

```diff
@@ -340,13 +351,13 @@ module.exports = class Dock {
   }
 
   getInitialSize () {
-    let initialSize
+    const items = this.paneContainer.getPaneItems()
     // The item may not have been activated yet. If that's the case, just use the first item.
-    const activePaneItem = this.paneContainer.getActivePaneItem() || this.paneContainer.getPaneItems()[0]
-    if (activePaneItem != null) {
-      initialSize = getPreferredSize(activePaneItem, this.location)
-    }
-    return initialSize == null ? DEFAULT_INITIAL_SIZE : initialSize
+    const activePaneItem = this.paneContainer.getActivePaneItem() || items[0]
+    let initialSize
+    if (activePaneItem) initialSize = getPreferredSize(activePaneItem, this.location)
+    if (initialSize == null && items.length) initialSize = DEFAULT_INITIAL_SIZE
+    return initialSize ? Math.max(MINIMUM_SIZE, initialSize) : null
```

Comment:
I gotta remove this `Math.max()` call. It's gonna be done in `render()` so this is redundant.